vincentlanglet/twig-cs-fixer/0.6/.twig-cs-fixer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+$ruleset = new TwigCsFixer\Ruleset\Ruleset();
+
+// You can start from a default standard
+$ruleset->addStandard(new TwigCsFixer\Standard\TwigCsFixer());
+
+//// And then add/remove/override some rules
+//$ruleset->addRule(new TwigCsFixer\Rules\File\FileExtensionRule());
+//$ruleset->removeRule(TwigCsFixer\Rules\Whitespace\EmptyLinesRule::class);
+//$ruleset->overrideRule(
+//    new TwigCsFixer\Rules\Punctuation\PunctuationSpacingRule(
+//        ['}' => 1],
+//        ['{' => 1],
+//    )
```

Comment:
Those classes doesn't exists in 0.6 version cf
https://github.com/VincentLanglet/Twig-CS-Fixer/tree/0.6.0/src

I don't think it's worth to have a recipe for < 3.0.0 since I only maintain the latest major.
So it seems better to only create
```
vincentlanglet/twig-cs-fixer/3.0/
```
files, no ?